From bf76c51f8db9950abd4dc2dfe12520c6c494366d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?V=C3=ADt=20Ondruch?= <vondruch@redhat.com>
Date: Tue, 13 Sep 2016 10:30:37 +0200
Subject: [PATCH] Drop 'hp' from compute tests.

---
 tests/compute/helper.rb | 16 ----------------
 1 file changed, 16 deletions(-)

diff --git a/tests/compute/helper.rb b/tests/compute/helper.rb
index 531a192..d477f53 100644
--- a/tests/compute/helper.rb
+++ b/tests/compute/helper.rb
@@ -67,14 +67,6 @@ def compute_providers
       },
       :mocked => false
     },
-    :hp       => {
-      :server_attributes => {
-        :flavor_id => 100,
-        :image_id  => 1242,
-        :name      => "fog_#{Time.now.to_i}"
-      },
-      :mocked => true
-    },
     :ibm => {
       :server_attributes => {},
       :mocked => true
@@ -82,14 +74,6 @@ def compute_providers
     :joyent => {
       :mocked => false
     },
-    :hp       => {
-      :server_attributes => {
-        :flavor_id => 100,
-        :image_id => 1242,
-        :name     => "fog_#{Time.now.to_i}"
-      },
-      :mocked => true
-    },
     :ninefold   => {
       :mocked => false
     },
-- 
2.10.0

