From 5fb2f60feb0e3562f2e18d8061027c740dd8a4aa Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?V=C3=ADt=20Ondruch?= <vondruch@redhat.com>
Date: Tue, 13 Sep 2016 10:47:33 +0200
Subject: [PATCH] Drop 'hp' from storage tests.

---
 tests/storage/helper.rb | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/tests/storage/helper.rb b/tests/storage/helper.rb
index fe9afd2..ec609af 100644
--- a/tests/storage/helper.rb
+++ b/tests/storage/helper.rb
@@ -3,9 +3,6 @@ def storage_providers
     :aws        => {
       :mocked => true
     },
-    :hp         => {
-      :mocked => true
-    },
     :internetarchive => {
       :mocked => true
     },
-- 
2.10.0

